docs/src/docs/dates/date-picker.mdx

```diff
@@ -107,6 +107,9 @@ function Demo() {
 `dateParser` function should always return Date object.
 If parsed date is invalid when input is blurred value will be restored to last valid value.
 
+Note that if you use the default dayjs parser, you will need to import and configure the
```

Comment:
Documentation should be included on DatePicker page, not index page, other components do not have dates parsing

Reply:
Ok, so I should remove the details about plugins entirely from the "Getting started" page and just put them inline here?

Reply:
yes